service/matching/db.go

```diff
@@ -46,26 +46,33 @@ import (
 const (
 	initialRangeID     = 1 // Id of the first range of a new task queue
 	stickyTaskQueueTTL = 24 * time.Hour
+
+	userDataEnabled userDataEnabledState = iota
```

Comment:
nit: userDataState instead of userDataEnabledState

Reply:
ok